app/controllers/concerns/collection.rb

```diff
@@ -8,10 +8,17 @@ module Collection
 
   included do
     def resolve_collection
-      result = sort(search(policy_scope(resource)))
+      result = filter_by_tags(sort(search(policy_scope(resource))))
       result.paginate(page: pagination_offset, per_page: pagination_limit)
     end
 
+    def filter_by_tags(data)
+      tags = params[:tags]
+      return data if !resource.column_names.include?('tags') || tags.empty?
+
+      data.where('tags @> ?', tags.to_json)
```

Comment:
Observation: `tags` could start to be considered as a common global field on any model that would use them. Since this only checks for the field presence, it would have to be become a standard. Otherwise any table/model having `tags` _not_ as JSON blob would break on this lookup.

I'm wondering where we can document this...

Reply:
It is documented by the insights API, they have a very strict specification about how a tag should look like :thinking:

Reply:
Okay, so what are you suggesting here? :thinking:

Reply:
Just to document it, that when a model with `tags` is exposed (has a controller) that the tags will follow the Insights tag structure.